src/chains/optimism/eips.ts

```diff
@@ -0,0 +1,29 @@
+import { EIP } from '@/types/eip';
+import { eip1559 as eip1559OnEthereum, eips as ethereumEIPs } from '../mainnet/eips';
+
+const eip1559OnOptimism: EIP = {
```

Comment:
Ah sorry we'll also need to change the hardforks for optimism. Regolith was the first post-bed rock hard fork live on mainnet, and 1559 has been live since Regolith. Since then Canyon has gone live, and Delta is coming soon (so the enum would just need Regolith and Canyon for now)

Reply:
Done in [6f4a973](https://github.com/mds1/evm-diff/pull/53/commits/6f4a973577d1032e7838c0ceeb6ff6c42742ab6b) and [5a06e9c](https://github.com/mds1/evm-diff/pull/53/commits/5a06e9cfbe9b17bd24b9d10e562eab96bf7adb77)